### PR TITLE
Adding an API description

### PR DESCRIPTION
**Pull request by hgaard**
New section describing how providers get consumers/partners started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,27 +11,26 @@ Over time the use cases for the platform is expected to expand to:
...
 
+## Providers
 
-# Providers
-
-As a provider to logic you can take advantage of the advanced service infrastructure we already operate, like identity/access management, logging and auditing or you can also just choose to expose your existing API's as is on the Logic Marketplace. If you want to learn more about how to get started as a provider [this](providers/getting-started.md) article will take you through a step by step guide on getting setup and approved as a provider.
-
+As a provider to logic you can take advantage of the advanced service infrastructure we already operate, like identity/access management, logging and auditing or you can also just choose to expose your existing API's as is on the Logic Marketplace. If you want to learn more about how to get started as a provider [this](./providers/getting-started.md) article will take you through a step by step guide on getting setup and approved as a provider. To learn more about submitting API's to the Marketplace, go [here](./providers/register-an-api.md). General information for providers are complied [here](./providers/readme.md)
```
Comment (base2browns) [inline README.md]: are complied => is available?

File: providers/register-an-api.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# How to register an API user for a customer
+
+This document describes how a consumer can gain access to an API exposed on the Logic Marketplace.
+
+## 1. Access request from the customer
+
+In order to get access to Logic, the consumer needs to be able to access the [Logic Console](console). Today there are 2 different ways to achieve this - setting up a federation with Logic Identity and the consumers own identity store (company or municipal AD/AAD) or requesting to have a user created in the Logic Console user store. Fro both, the consumer needs to contact [support](mailto:logic-admin@kmd.dk) with the following information.
```
Comment (base2browns) [inline providers/register-an-api.md]: Fro = For

File: providers/register-an-api.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+- Company name
+- CVR number of the company
+- Contact information:
+  - Full name of the contact person
+  - Email address
+  - Phone number
+  - Company address
```
Comment (base2browns) [inline providers/register-an-api.md]: We only need the email & phone number right now

File: providers/register-an-api.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+## 1. Create a Logic Subscription
+
+When the consumer/partner is setup with access to the console the consumer will now have to setup a Logic subscription through the [Logic Console](console)
+  
+1. Go to [Logic Console](console)
+2. Choose `Subscriptions` from the menu and then click on `Add Subscription`
+3. Enter a `Subscription Name` of the partners name. A `Starter` subscription is fine. Click on `Create`
```
Comment (base2browns) [inline providers/register-an-api.md]: You can remove the reference to Starter now

File: providers/register-an-api.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+4. Record the `Subscription Id`
+
+You will now have a Logic Subscription. If you want to setup multiple subscriptions for better tracking, you jsn just go ahead and create more and you can switch between subscriptions by returning to the `Subscriptions` page and clicking `Activate` on the subscription you wish to manage.
```
Comment (base2browns) [inline providers/register-an-api.md]: jsn

File: providers/register-an-api.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+## 1. Arrange for a Partner/Consumer Access and Client Credentials
+
+To allow the consumer/partner to interact with [Logic Console](console) they will need to be registered for access. This can either be:
+
+- By adding their Active Directory as a downstream provider for [Logic Console](console) (a good option for larger organisations)
+- Adding users to the `Logic Console Users` AAD which is managed by the Logic team
+
+If your API uses Logic Identity for authentication each partner might also require a `Client Credential` which will allow them to programmatically call your API.
+
+Go to the [Logic Issues Dashboard](https://dev.azure.com/kmddk/KMDLoGIC/_dashboards/dashboard/02f46530-28bb-4962-bc2a-a12eabfe383e) and use the `New Work Item` widget to create your request.
```
Comment (base2browns) [inline providers/register-an-api.md:35]: Is this OK to publish here?

**Review by base2browns — CHANGES_REQUESTED** [review]

File: providers/register-an-api.md (hunk 6 above)
Comment (hgaard) [reply providers/register-an-api.md:35]: I think so (for now). Might be sub optimal for an external that they can get access. But there is no other good way just now

File: providers/register-an-api.md (hunk 5 above)
Comment (hgaard) [reply providers/register-an-api.md]: ?

Comment (hgaard) [reply providers/register-an-api.md]: sorry too quick :)

File: providers/register-an-api.md (hunk 4 above)
Comment (hgaard) [reply providers/register-an-api.md]: Good point. Thanks

**Pull request merged**